src/builder.js

```diff
@@ -21,13 +21,24 @@ import IdentityManager from './services/identity_manager';
 import {connectToMongo} from './utils/db_utils';
 import HttpsClient from './utils/https_client';
 import TokenAuthenticator from './utils/token_authenticator';
-import {createWeb3} from './utils/web3_tools';
+import {createWeb3, getDefaultAddress} from './utils/web3_tools';
+import RolesRepository from './services/roles_repository';
 
 class Builder {
   async ensureAdminAccountExist() {
     await this.dataModelEngine.addAdminAccount();
   }
 
+  async ensureAccountIsOnboarded() {
```

Comment:
This method should accept a list of expected roles, and complain if the actual role is not on it. This will be used as soon as when we implement workers.